Task_TheFlexGame/index.css

```diff
@@ -1 +1,59 @@
-/* Write code below for FLEX GAME TASK */
+<html>
+    <head>
+        <title>The Flex Game</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="style.css">
+        <style>
+   
+            .dash-container{
+            display:grid;
+             height:100%;
+             width:100%;
+            grid-template-columns:20% 80%;
+            grid-template-rows:20% auto;
+            }
+           .dash-item{ 
+            color:#FFFF;
+            font-size:0.9em;
+            margin-left:1em;
+            margin-top:1em;
+            }
+            .card-container{
+            display:grid;
+            grid-template-columns: repeat(auto-fit, minmax(110px, 1fr));
+            width:80%;
+            margin-left:1em;
+            }
+            .card-item{
+             width:100px;
+             height:100px;

```

Comment:
Consider using a consistent unit (e.g., 'px') for margin values throughout the CSS for uniformity.